Task1_KFox/app/index.html

```diff
@@ -7,142 +7,187 @@
-                <div class="recentlyPlayed-cover"></div>
-                <div class="recentlyPlayed-author">Lynyrd Skynyrd</div>
-                <div class="recentlyPlayed-title">Free Bird</div>
-                <div class="recentlyPlayed-length">3:31</div>
-            </div> 
-            <div class="recentlyPlayed-item">
-                <div class="recentlyPlayed-cover"></div>
-                <div class="recentlyPlayed-author">Led Zeppelin</div>
-                <div class="recentlyPlayed-title">Stairway to Heaven</div>
-                <div class="recentlyPlayed-length">5:36</div>
-            </div>            
-            <div class="recentlyPlayed-item">
-                <div class="recentlyPlayed-cover"></div>
-                <div class="recentlyPlayed-author">The Who</div>
-                <div class="recentlyPlayed-title">Won't Get Fooled Again</div>
-                <div class="recentlyPlayed-length">4:36</div>
-            </div>            
-            <div class="recentlyPlayed-item">
-                <div class="recentlyPlayed-cover"></div>
-                <div class="recentlyPlayed-author">The Rolling Stones</div>
-                <div class="recentlyPlayed-title">You Can't Always Get What You Want</div>
-                <div class="recentlyPlayed-length">9:16</div>
-            </div>            
-            <div class="recentlyPlayed-item">
-                <div class="recentlyPlayed-cover"></div>
-                <div class="recentlyPlayed-author">Guns N'Roses</div>
-                <div class="recentlyPlayed-title">Patience</div>
-                <div class="recentlyPlayed-length">6:11</div>
-            </div>            
-            <div class="recentlyPlayed-item">
-                <div class="recentlyPlayed-cover"></div>
-                <div class="recentlyPlayed-author">Cinderella</div>
-                <div class="recentlyPlayed-title">Don't Know What You Got (Till It's Gone)</div>
-                <div class="recentlyPlayed-length">4:39</div>
+        <div id="landingPage">
+            <header>                
+                <img id="hamburger" src="images/hamburger.png" alt="">
+                <img id="logo" src="images/logotype.png" alt="">
+                <img id="volume" src="images/volume.png" alt="">
+            </header>            
+            <div id="np-panel">
+                <img id="np-panel-cover" src="images/nowplaying_image.png" alt=""/>
+                <p>Jimmi Hendrix Experience</p>
+                <p>Long Hot Summer Night</p>
+                <div id="np-panel-controls">
+                    <img id="np_stop" src="images/np_controlls_stop.png" alt="" >
+                    <img id="np_play" src="images/np_controlls_play.png" alt="" >
+                    <img id="np_next" src="images/np_controlls_next.png" alt="" >                    
+                </div>
             </div>
-            
         </div>
-        <div id="concerts">
+        <section id="recentlyPlayed">
```

Comment:
For future, please always use lisp-case or other naming convention.
For more details: http://stackoverflow.com/questions/7927193/css-class-naming-convention